src/AlertsManagement/AlertsManagement/Microsoft.Azure.PowerShell.Cmdlets.AlertsManagement.generated.format.ps1xml

```diff
@@ -140,11 +140,11 @@
             <ListItems>
               <ListItem>
                 <PropertyName>GroupBy</PropertyName>
-                <Label>TotalSmartGroups</Label>
```

Comment:
This needs a changelog entry

Reply:
What kind of entry? This would be the first time this module will be published so there is no previous reference. Am I right?
Moreover, it is nothing like voluntary change in label name but more like error while development.

Reply:
@mahakjain314 that's correct -- since this module hasn't been released previously, there shouldn't be a change log entry needed for this PR. I'll resolve this comment.